task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -63,8 +67,9 @@ spec:
         image_with_digest="${IMAGE_URL}@${IMAGE_DIGEST}"
 
         if ! oc image extract "${image_with_digest}" ; then
-          echo "Unable to extract image! Skipping checking binaries!"
-          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'Unable to extract image! Skipping checking binaries!')"
+          echo "Unable to extract image, unable to validate extracted binaries of image!"
```

Comment:
SR if exposed to user: echo "Unable to extract image or validate extracted binaries."